src/Tribe/Views/V2/Assets.php

```diff
@@ -419,7 +423,7 @@ public function register() {
 				'priority'     => 10,
 				'conditionals' => [ $this, 'should_enqueue_frontend' ],
 				'groups'       => [ static::$group_key, static::$widget_group_key ],
-				'in_footer'    => false,
+				'in_footer'    => array( 'strategy'  => 'defer' ),
```

Comment:
`in_footer` is [expected to be a boolean](https://github.com/the-events-calendar/tribe-common/blob/master/src/Tribe/Assets.php#L579). I don't believe this is doing the defer as you intended (unless I'm missing something).

Reply:
You may need a corresponding PR in `common`.

Reply:
right, common will need updating to make this work. I will update the PR there to include a doc block update.

Reply:
I think the functionality add is worth continuing forward as it is.

Eventually it would be nice to rename the `in_footer` param to more closely align with the `wp_register_script` docs, to avoid confusion. But I don't think that necessarily has to be done here, it could be a bit of a lift to do that now. 

We are essentially the `$args` now, not just `$in_footer` bool.

``` 
@param array|bool       $args     {
 *      Optional. An array of additional script loading strategies. Default empty array.
 *      Otherwise, it may be a boolean in which case it determines whether the script is printed in the footer. Default false.
 *
 *      @type string    $strategy     Optional. If provided, may be either 'defer' or 'async'.
 *      @type bool      $in_footer    Optional. Whether to print the script in the footer. Default 'false'.
 * }
 ```
 
Lookin good!

Reply:
I updated the common PR based on your feedback, please let me know if I missed anything!